src/java/org/apache/cassandra/cql3/functions/JavaBasedUDFunction.java

```diff
@@ -591,14 +592,17 @@ private NameEnvironmentAnswer findType(String className)
 
             String resourceName = className.replace('.', '/') + ".class";
 
-            try (InputStream is = UDFunction.udfClassLoader.getResourceAsStream(resourceName))
+            try
```

Comment:
Not sure why we need a nested `try` block here.

Reply:
Ah - I've changed that part twice. But yea, one try-catch is absolutely sufficient.